pkg/kubectl/cmd/set/set_serviceaccount.go

```diff
@@ -124,9 +125,12 @@ func (saConfig *serviceAccountConfig) Complete(f cmdutil.Factory, cmd *cobra.Com
 	}
 	saConfig.serviceAccountName = args[len(args)-1]
 	resources := args[:len(args)-1]
+	includeUninitialized := cmdutil.GetIncludeUninitialized(cmd, saConfig.all, "")
+	cmdutil.GetExplicitIncludeUninitialized(cmd, &includeUninitialized)
```

Comment:
Is `GetExplicitIncludeUninitialized` already invoked in `GetIncludeUninitialized`?